### ISSUE TITLE
Error ITMS-90035 when submitting IOS app

### ISSUE THREAD

**Issue by Michoko92** [airsdk/Adobe-Runtime-Support#1276]
Hi,

I'm facing an issue I can't solve for now. I try to update some iOS apps with latest AIR SDK 33.1.1.633 (on Windows), but when I try to upload the IPA with Transporter in my MacOS VM, it get a "ERROR ITMS-90035: Invalid Signature. A sealed resource is missing or invalid. The file is not properly signed." error.

My adt command line is:
`
call adt -package -target ipa-app-store -useLegacyAOT no -keystore "..\..\..\Certificats\CertificatsDistrib.p12" -storetype pkcs12 -storepass XXX -provisioning-profile "keys\AOK_Store.mobileprovision" ApplicationAOK.ipa application.xml AOK.swf icons\iOS Default@2x~iphone.png Default-568h@2x~iphone.png Default-375w-667h@2x~iphone.png Default-414w-736h@3x~iphone.png Default-812h@3x~iphone.png Default-896h@2x~iphone.png Assets Assets.car LaunchScreen.storyboardc -extdir ..\..\..\Libs\extensions`

I regenerated my provisioning files several times, just to be sure. My certificate only expires in September 2022. Debug and Adhoc apps work fine, even on my iOS 15 device. However, when I try to install the final app store IPA to my iOS 15 device, I get this error: 

```
WARNING: could not locate iTunesMetadata.plist in archive!
WARNING: could not locate Payload/Angels.app/SC_Info/Angels.sinf in archive!
ERROR: Install failed. Got error "ApplicationVerificationFailed" with code 0xe8008015: Failed to verify code signature of /var/installd/Library/Caches/com.apple.mobile.installd.staging/temp.IOa5tt/extracted/Payload/Angels.app : 0xe8008015 (A valid provisioning profile for this executable was not found.)
```

But I suppose it is normal since it is the App Store build, right? However, when I try to upload this final IPA, I get the error above,  and I really can't figure out why. It always worked fine before... I checked everything many times, but I can't find what's wrong. Would you have a suggestion please?

**Comment by Michoko92** [comment]
Just a bit more information: I tried with another app, using same distribution certificate, and it works! (the build is sent via Transporter). The ADT commandline is almost identical between the two apps (the only difference is that the non-working app includes a custom storyboard). The only explanation I can see would be that the provisioning profile is incorrect, but I keep checking it, and it is indeed a valid App Store provisioning profile. I'm really puzzled... Thank you.

**Comment by Michoko92** [comment]
Hi Andrew, thank you so much for your support. I sent an IPA, and can send more if needed. Cheers! :)

**Comment by ajwfrost** [comment]
That one looks fine from what we can see, extracted on a mac and checking the code signature...
Are you able to also provide one that used the same certificate and provisioning profile so that we can compare them?
thanks

**Comment by Michoko92** [comment]
Okaaay, now this is interesting. Before sending you the working app, I wanted to generate it again, just to be sure. And once I generated it, it didn't work in Transporter too! The only thing I changed since last publishing (of the 15th of October) was the storyboard file. I removed it from the build, and indeed it worked again in Transporter.

Then I spotted the culprit: in the "LaunchScreen.storyboardc" I had copied from my Mac VM, it had created 3 extra metadata files, named: 
._01J-lp-oVM-view-Ze5-6b-2t3.nib
._Info.plit
._UIViewController-01J-lp-oVM.nib

Once I deleted them, only keeping the 3 useful files, everything worked again! Don't ask me why 3 files with special names could make Transporter think that the IPA is not properly signed, but this is what happens. I already saw similar posts on forums, where people using special characters in their file names had the same error, and it seems that files named like above are also causing the issue.

Thank you so much for your help and for the moral support! :)
